docs/03-tutorials/00-serverless/svls-03-expose-function.md

```diff
@@ -150,19 +150,19 @@ Follow these steps:
 
 1. Select a Namespace from the drop-down list in the top navigation panel. Make sure the Namespace includes the Function that you want to expose through an APIRule.
 
-2. Go to **Workloads** > **Functions** and select the Function you want to expose.
+2. Go to **Discovery and Network** > **API Rules**, and click on **Create API Rule**.
 
-3. Switch to the **Configuration** tab and select **Create API Rule** in the API Rules section.
-
-4. Under **General settings**, enter the following information:
+3. Enter the following information:
 
     - The APIRule's **Name** matching the Function's name.
 
     >**NOTE:** The APIRule CR can have a name different from that of the Function, but it is recommended that all related resources share a common name.
 
-    - **Subdomain** to determine the host on which you want to expose your Function.
+    - **Service Name** matching the Function's name.
+
+    - **Host** to determine the host on which you want to expose your Function. It is required to change the `*` symbol at the beggining to the subdomain name you want.
 
-5. In the **Rules** section, select the `noop` handler and mark **all** the methods.
+5. In the **Rules > Access Strategies > Config**  section, change the handler from `allow` to `noop` and select **all** the methods below.
```

Comment:
Does it mean that user should pick up all methods one by one? I don't see the *all* option.

Reply:
yes

Reply:
```suggestion
5. In the **Rules > Access Strategies > Config**  section, change the handler from `allow` to `noop` and select all the methods below.
```